rust-bins/wallet-notes/example.c

```diff
@@ -139,6 +148,12 @@ int main(int argc, char *argv[]) {
       out = combine_encrypted_amounts(argv[2], argv[3], &flag);
       printf("%s\n", out);
       return (int)flag;
+    } else if (strcmp(argv[1], "generate-baker-keys") == 0) {
+      uint8_t flag = 1;
+      char *out;
+      out = generate_baker_keys(&flag);
+      printf("%s\n", out);
+      return (int)flag;
```

Comment:
Why not use `return printStr(out, flag);`?

Reply:
Using `return printStr(out, flag);` now.